app/helpers/search_helper.rb

```diff
@@ -85,6 +85,20 @@ def ordering
     end
   end
 
+  # Transforms controller params from get request into query args for a search
+  # @return [Hash]
+  def get_query_args
```

Comment:
Post is not working out so well, I am having issues re-encoding the binary and downloading it in memory after making the post request, this is also causing the page to freeze while it converts to blob. Do you have any suggestions on how to do a post while avoiding client-side js? A form post may also be possible but cannot be nested in the existing form.